coretex/cli/commands/node.py

```diff
@@ -114,6 +115,7 @@ def update() -> None:
             return
 
     node_module.stop()
+    node_module.removeOldImage()
```

Comment:
Run image cleanup after node has started. Clean all images with the same repo as the image which was pulled last. Do not remove the image which is going to be used for starting the node